specification/msi/resource-manager/Microsoft.ManagedIdentity/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -0,0 +1,544 @@
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The requested identity.",
+            "schema": {
+              "$ref": "#/definitions/Identity"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "UserAssignedIdentities_Delete",
+        "description": "Deletes the identity.",
+        "x-ms-examples": {
+          "IdentityDelete": {
+            "$ref": "./examples/IdentityDelete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Deleted Identity.",
+            "schema": {
+              "$ref": "#/definitions/Identity"
+            }
+          },
+          "204": {
+            "description": "The specified identity does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Identity": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The id of the created identity."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the created identity."
+        },
+        "location": {
+          "type": "string",
+          "description": "The Azure region where the identity lives."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties associated with the identity.",
+          "$ref": "#/definitions/IdentityProperties"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Microsoft.ManagedIdentity/userAssignedIdentities"
```

Comment:
why is this not just userAssignedIdentities. Do you need the namespace also in the value?

Reply:
hmm.. not really. This pattern emerged from implicit identity where type is set to the resource that owns the identity. For example in case of implicit identity for VM, type would be Microsoft.Compute/VM_Name. We are following the same pattern for explicit identity.

Reply:
I think this is OK as well